src/cmd/flux-jobs.py

```diff
@@ -89,54 +96,28 @@ def output_format(fmt, jobs):
         print(s)
 
 
-@flux.util.CLIMain(logger)
-def main():
-    parser = argparse.ArgumentParser()
-    # -a equivalent to -s "pending,running,inactive" and -u set to userid
-    parser.add_argument(
-        "-a", action="store_true", help="List all jobs for current user"
-    )
-    # -A equivalent to -s "pending,running,inactive" and -u set to "all"
-    parser.add_argument("-A", action="store_true", help="List all jobs for all users")
-    parser.add_argument(
-        "-c",
-        "--count",
-        type=int,
-        metavar="N",
-        default=1000,
-        help="Limit output to N jobs(default 1000)",
-    )
-    parser.add_argument(
-        "-s",
-        "--states",
-        type=str,
-        metavar="STATES",
-        default="pending,running",
-        help="List jobs in specific states(pending,running,inactive)",
-    )
-    parser.add_argument(
-        "--suppress-header",
-        action="store_true",
-        help="Suppress printing of header line",
-    )
-    parser.add_argument(
-        "-u",
-        "--user",
-        type=str,
-        metavar="[USERNAME|UID]",
-        default=str(os.geteuid()),
-        help='Limit output to specific username or userid (Specify "all" for all users)',
-    )
-    parser.add_argument(
-        "-o",
-        "--format",
-        type=str,
-        metavar="FORMAT",
-        help="Specify output format using Python's string format syntax",
-    )
+def fetch_jobs_stdin(args):
+    """
+    Return a list of jobs gathered from a series of JSON objects, one per
+    line, presented on stdin. This function is used for testing of the
+    flux-jobs utility, and thus, all filtering options are currently
+    ignored.
+    """
+    import fileinput
+    import json
+
+    jobs = []
+    for line in fileinput.input("-"):
+        try:
+            job = json.loads(line)
+        except Exception as e:
```

Comment:
I'm surprised pylint didn't complain about this broad exception block.  Since the next action is to log an error and exit, it is actually probably fine to leave this the way it is.   FWIW (since I already looked it up), the py3 docs say that this method will throw `JSONDecodeError` [(source)](https://docs.python.org/3/library/json.html#json.loads).  Feel free to leave this the way it is though.

Reply:
I don't think we're running pylint on commands :rofl: - so I should probably fix this!

Reply:
Would it be acceptable to catch `ValueError` instead?

Reply:
as an aside since you're here, I think `lgtm` will flag the single character variable `e`.

Reply:
pylint, yeah. Though its annoying I'll fix that too.